crates/ra_db/src/input.rs

```diff
@@ -76,6 +76,7 @@ impl SourceRoot {
 #[derive(Debug, Clone, Default, PartialEq, Eq)]
 pub struct CrateGraph {
     arena: FxHashMap<CrateId, CrateData>,
+    env: FxHashMap<String, String>,
```

Comment:
That should be per-crate, and not per CrateGraph. That is, it belongs to `CrateData`

Reply:
Thanks, will propagate that map further down!